packages/core/sample-projects/Development.md

```diff
@@ -0,0 +1,25 @@
+# Development
+
+## Environment
+
+You need to start a redis service. And the redis's default port is 6379. You can edit it at `kuai.config.ts`
+
+```typescript
+import { KuaiConfig } from '@ckb-js/kuai-core';
+
+const config: KuaiConfig = {
+  REDIS_PORT: 6379,
+};
+
+export default config;
+```
+
+## Quick Start
+
+```sh
+git clone
```

Comment:
Is `git clone` a complete command here? And is there a `cd` command after `git clone`?

Reply:
Because it depends on the new project name, it will fill with the project name that creates by `kuai init`. 
It may need to be improved later by a template.

Reply:
Placeholder as `<project name>` could be used to imply the parameter, similar to 
```
# To download a file:
curl <url>

# To download and rename a file:
curl <url> -o <outfile>

# To download multiple files:
curl -O <url> -O <url>

# To download all sequentially numbered files (1-24):
curl http://example.com/pic[1-24].jpg

# To download a file and pass HTTP authentication:
curl -u <username>:<password> <url>

# To download a file with a proxy:
curl -x <proxy-host>:<port> <url>

# To download a file over FTP:
curl -u <username>:<password> -O ftp://example.com/pub/file.zip

# To get an FTP directory listing:
curl ftp://username:password@example.com

# To resume a previously failed download:
curl -C - -o <partial-file> <url>

# To fetch only the HTTP headers from a response:
curl -I <URL>
```